mrp_unbuild_advanced/models/mrp_unbuild.py

```diff
@@ -39,6 +40,11 @@ class MrpUnbuild(models.Model):
         comodel_name="mrp.tag",
     )
 
+    @api.onchange("shift_start_date", "shift_end_date")
+    def _onchange_shift_start_date_shift_end_date(self):
+        if self.shift_end_date and self.shift_start_date:
```

Comment:
`else` statement is missing (e.g. if a date is not filled, set total time to 0.0

Reply:
Updated